lambda_functions/create_rds_iam_user/lambda-handler.py

```diff
@@ -21,10 +21,13 @@
 
 if TYPE_CHECKING:
     from mypy_boto3_iam import IAMClient
+    from mypy_boto3_sts import STSClient
 
-    client: IAMClient = boto3.client("iam")
+    boto3_iam_client: IAMClient = boto3.client("iam")
+    boto3_sts_client: STSClient = boto3.client("sts")
 else:
-    client = boto3.client("iam")
+    boto3_iam_client = boto3.client("iam")
+    boto3_sts_client = boto3.client("sts")
```

Comment:
It's a bit dangerous to duplicate the functionality here, since it might start diverging. Instead, it's possible to only import the type annotation itself, and do the annotation outside the `if` statement.